docs/tutorial.rst

```diff
@@ -118,129 +118,118 @@ The Python interface
+    >>> species = stdpopsim.get_species("homsap")
 
-.. .. code-block:: python
+3. Set the contig length. We are simulating 0.1 x chromosome 22,
+which is about 5Mb. Again, you could use a fraction of any of the
+chromosomes listed in the :ref:`sec_catalog`, keeping in mind that
+larger contigs will take longer to simulate.
 
-..     import stdpopsim
-..     from stdpopsim import homo_sapiens
+.. code-block:: python
 
-..     chrom = homo_sapiens.genome.chromosomes["chr22"]
-..     model = homo_sapiens.GutenkunstThreePopOutOfAfrica()
-..     # One sample each from YRI, CEU and CHB.
-..     samples = [msprime.Sample(population=j, time=0) for j in range(3)]
-..     ts = msprime.simulate(
-..         samples=samples,
-..         recombination_map=chrom.recombination_map(),
-..         mutation_rate=chrom.default_mutation_rate,
-..         **model.asdict())
+    >>> contig = species.get_contig("chr22", length_multiplier=0.1)
 
-.. (This simulation now has a realistic effective population size,
-.. so will produce thousands of variant sties, but still runs very fast.)
+4. Set the model as the generic piecewise constant size model, using the
+predefined human effective population size (see :ref:`sec_catalog`).
+Since we are providing one effective population size, the model is constant
+population size for one population over time.
 
+.. code-block:: python
 
+    >>> model = stdpopsim.PiecewiseConstantSize(species.population_size)
 
-.. _sec_tutorial_generic_models:
+5. Set the number of samples and set the simulation engine.
+In this case we will simulate 10 samples and use the default simulator,
+which is `msprime`. But, you can go crazy with the sample size!
+`msprime` is great at simulating large samples!
 
-**************
-Generic models
-**************
+.. code-block:: python
 
-The previous sections showed how to simulate population genetic models that have
-been published for particular species. It is also sometimes useful to simulate
-more generic models. We can still use the information for particular species to
-do this; the only difference is that we instantiate the models directly rather
-than retrieving them from the :ref:`catalog <sec_catalog>`.
+    >>> samples = model.get_samples(10)
+    >>> engine = stdpopsim.get_default_engine()
 
+6. Simulate the model with the contig length and number of samples we defined above.
+We capture the simulation results in a `tree sequence object <https://tskit.readthedocs.io/en/stable/python-api.html#tskit.TreeSequence>`_.
 
 .. code-block:: python
 
-    species = stdpopsim.get_species("homsap")
-    contig = species.get_contig("chr22", length_multiplier=0.1)
-    model = stdpopsim.PiecewiseConstantSize(species.population_size)
-    samples = model.get_samples(10)
-    engine = stdpopsim.get_default_engine()
-    ts = engine.simulate(model, contig, samples)
+    >> ts = engine.simulate(model, contig, samples)
 
+7. We can now do some simple checks that our simulation worked with ``tskit``.
```

Comment:
Maybe link to the tskit docs here: tskit.readthedocs.io

Reply:
Can I make a direct link?

I was going to do:
```
``tskit``
(https://tskit.readthedocs.io)
```

Reply:
It's something like `tskit <https://tskit.readthedocs.io>`_

Reply:
Gah, trying to do RST syntax in markdown is a pain! See the second example here: http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets

Reply:
ok cool.

and just make a big code block, then you can have the RST syntax.